easybuild/easyconfigs/s/STELLA/STELLA-1.05.00-CrayGNU-17.08-double.eb

```diff
@@ -0,0 +1,37 @@
+# jg (cscs)
+easyblock = 'CMakeMake'
+
+name = 'STELLA'
+version = '1.05.00'
+versionsuffix = '-double'
+
+homepage = 'https://github.com/MeteoSwiss-APN/stella'
+description = """STELLA"""
+
+toolchain = {'name': 'CrayGNU', 'version': '17.08'}
+toolchainopts = {'verbose': False}
+# can't clone private c2sm repo (+ no official release)
+sources = [
+    {'filename': '/apps/common/UES/easybuild/sources/s/%(name)s/%(namelower)s-%(version)s.tar.gz'},
+]
+
+builddependencies = [ ('CMake', '3.11.4', '', True), ]
+
+dependencies = [ 
+    ('Serialbox', '2.3.1'),
+    ('Boost', '1.65.0'),
+    ('craype-accel-nvidia60', EXTERNAL_MODULE),
+]
+
+configopts  = "-DBoost_INCLUDE_DIR=${EBROOTBOOST}/include "
```

Comment:
configopts += "-DSTELLA_BITREPRO=OFF "

I needed to include this, otherwise the COSMO build later would fail.